### PR TITLE
Cleaned up AC auto plugin enable and added rF2 auto plugin enable.

### PR DESCRIPTION
**Pull request by Sparten**

### CODE REVIEW

File: CrewChiefV4/PluginInstaller.cs [hunk 1]
```diff
@@ -230,52 +257,49 @@ public void InstallOrUpdatePlugins(GameDefinition gameDefinition)
...
-                                        if (lines[i + 1].Equals("ACTIVE=0"))
-                                        {
-                                            lines[i + 1] = "ACTIVE=1";
-                                        }
-                                        found = true;
-                                        break;
-                                    }
-                                }    
+                                //the whitespace is intended, this is how the game writes it.
+                                plugins[rf2PluginFileName][" Enabled"] = 1;
```
Comment (TheIronWolfModding) [inline CrewChiefV4/PluginInstaller.cs]: is it possible to add sanity check in this function, to make sure plugin is already on a disk and in a right location?  We're changing user's json, and I feel like we should be extra careful.  Also, what about saving backup of json and even .dlls?

Comment (Sparten) [inline CrewChiefV4/PluginInstaller.cs]: Why would you want to save the .dlls we dont delete anything other then our own dll, when it needs updating.
As to making sure that the plugin exists i can move the install/update function to the to of the if statement and that way make sure the dll is on disk.

Comment (TheIronWolfModding) [inline CrewChiefV4/PluginInstaller.cs]: You're right regarding dlls.  What about .jsons?  We update them silently, and user has no idea what we changed.  But I don't feel strongly about this, just my usual paranoia ;)

Comment (Sparten) [inline CrewChiefV4/PluginInstaller.cs]: I can add a message box asking if the user wants to auto enable the plugin incase its disabled or not present

File: CrewChiefV4/PluginInstaller.cs [hunk 2]
```diff
@@ -273,14 +285,26 @@ public void InstallOrUpdatePlugins(GameDefinition gameDefinition)
...
+                                }
                             }
                             else
                             {
-                                plugins.Add(rf2PluginFileName, new Dictionary<string, int>() { { " Enabled", 1 } });
+                                if (presentEnableMessagebox())
+                                {
+                                    plugins.Add(rf2PluginFileName, new Dictionary<string, int>() { { " Enabled", 1 } });
+                                    json = JsonConvert.SerializeObject(plugins, Formatting.Indented);
+                                    File.WriteAllText(configPath, json);
```
Comment (TheIronWolfModding) [inline CrewChiefV4/PluginInstaller.cs]: Nice, this is better :)  let's write to console that we updated .json/.ini at ...  Basically my biggest concern is us changing user files without them knowing, I also still think backup is good idea still (for config files), but we don't have to agree on everything ;)

**Pull request merged**